apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -235,6 +235,13 @@ export default class ModerationNotification extends Component {
       userImageDiv = <div className="user-avatar user-avatar--small user-avatar--shadow mb-1 userindicator__btn-img" style={sectionStyle} />
     }
 
+    let commentChangeLog
+    if (isModified === true) {
```

Comment:
would it work the same way if you omit `=== true` ?

Reply:
and from the condition would it rather be the other way around? so if `isModified` is true to show last edited otherwise created?